github/is-member-of/action.yml

```diff
@@ -24,13 +24,18 @@ runs:
     - id: gh-api-is-member
       name: Check if user is member of the given org
       run: |
-        if gh api \
+        # required because 'gh api' will return non-zero for 404s to indicate lack of membership
+        set +e
```

Comment:
[for reviewer] this was required otherwise the shell execution terminates, however it was not the case when wrapped in an `if` statement.